### ISSUE TITLE
[Question] xrmApp.Entity.ClickSubgridAddButton not working

### ISSUE THREAD

**Issue by Cornholio1337** [Microsoft/EasyRepro#716]
## Question

Hey guys. I already commented this to an older closed issue but to be safe I will open a new issue as well.
I need to click on the "add new" button in a UI subgrid. When you search for it on google you always find the function xrmApp.Entity.ClickSubgridAddButton, but when I try to use it, I get the error that Entity does contain a definition for ClickSubgridAddButton.
Is this function deprecated? And if so, is there any alternative?

Thanks in advance and greetings from Germany!

**Comment by Cornholio1337** [comment]
I tried to do it via client.Browser.Driver.FindElement(OpenQA.Selenium.By.Id(@elementname)); but it's not working because the Element Name of the Button changes a little bit every time you reload the page. 
Any suggestions?

**Comment by mikhailchub** [comment]
Hey!
There was a [question](https://github.com/microsoft/EasyRepro/issues/727), closed just a few hours ago. If your problem is similar to what was described there - you may take a look to the comments for the resolution suggestion. It may help.

**Comment by Cornholio1337** [comment]
> Hey!
> There was a [question](https://github.com/microsoft/EasyRepro/issues/727), closed just a few hours ago. If your problem is similar to what was described there - you may take a look to the comments for the resolution suggestion. It may help.

Awesome! Thanks for the reply. Will try it out right now.

**Comment by Cornholio1337** [comment]
Alright I was able to get it working. @mikhailchub your comment got me into the right direction.
In the end i used var elementAdd = client.Browser.Driver.FindElement(OpenQA.Selenium.By.XPath("//button[@aria-label='" + _name of the button_ +"']")); 
elementAdd.Click();
The button name I found via pressing F12,  right click on the button > Inspect and copy the name of the aria-label.
Thanks folks!
